app/views/ReportView/components/CopyNumber/index.tsx

```diff
@@ -26,8 +26,8 @@ const TITLE_MAP = {
   lowExp: 'Lowly Expressed Tumour Suppressors with Copy Losses',
 };
 
-const getInfoDescription = (relevance: string) => `Copy variants where the variant matched 1 or 
-more statements of ${relevance} relevance in the knowledge base matches section. Details on these 
+const getInfoDescription = (relevance: string) => `Copy variants where the variant matched 1 or
```

Comment:
Removing whitespace here affects the string itself so it should be kept

Reply:
must of been auto format